src/regression_tracker.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, configs, args):
     def _setup(self, args):
         return self._api_helper.subscribe_filters({
             'state': 'done',
-            'result': 'fail',
+            'name': 'checkout',
```

Comment:
I think it will still be useful to catch regressions earlier than when the top-level checkout node is done.

Reply:
Yes, that can also be done. Either we check one by one for every `done` node or look for all the nodes at once from top-level node, it will be the same at the end.

Reply:
In both cases, we will need a different set of preconditions e.g. in case we look for each node one by one, we need to have some checks to see if the node has failed and if the node is not checkout.
We should go with the approach that is better in terms of performance. Maybe we can compare the number of API calls.
Also, in the approach with `checkout` node, the script will track regression after the timeout has been reached and not as soon as it occurred. That is another downside.